src/Collection.php

```diff
@@ -65,6 +65,73 @@ public function __construct(Manager $manager, $ns, WriteConcern $wc = null, Read
     }
 
     /**
+     * Create a single index in the collection.
+     *
+     * @see http://docs.mongodb.org/manual/reference/command/createIndexes/
+     * @see http://docs.mongodb.org/manual/reference/method/db.collection.createIndex/
+     * @param array|object $keys
+     * @param array        $options
+     * @return string The name of the created index
+     */
+    public function createIndex($keys, array $options = array())
+    {
+        // TODO
+    }
+
+    /**
+     * Create multiple indexes in the collection.
+     *
+     * TODO: decide if $models should be an array of associative arrays, using
+     * createIndex()'s parameter names as keys, or tuples, using parameters in
+     * order (e.g. [keys, options]).
+     *
+     * @see http://docs.mongodb.org/manual/reference/command/createIndexes/
+     * @see http://docs.mongodb.org/manual/reference/method/db.collection.createIndex/
+     * @param array $models
+     * @return string[] The names of the created indexes
+     */
+    public function createIndexes(array $models)
+    {
+        // TODO
+    }
+
+    /**
+     * Drop this collection.
+     *
+     * @return Result
+     */
+    public function drop()
+    {
```

Comment:
```
$collection = $db->createCollection("foo", array("storageEngineOptions" => array("xyz"));
$collection->insert(array("foo" => "bar"));
$collection->drop();
$collection->insert(array("foo" => "bar")); // This will create different configured collection then above
```